### PR TITLE
Cache Bookie status on DB

### PR DESCRIPTION
**Pull request by eolivelli**
- Store information about Bookie on a table
- Handle status of the Bookie 'available'/'readonly'/'down'
- Clean up the 'cache' page and make it easier to use (still very ugly)
- Dump current bookkeeper client configuration
- Clean up the dependency to slf4j
- Drop mockRequest stuff and enable 'development server' on localhost:8086

### CODE REVIEW

**Comment by eolivelli** [comment]
This is only a draft, I have problems with getBookieInfo when a Bookie is down.
BookKeeper client seems to be stuck on waiting for Bookie connection, even when setting a 1 second timeout

File: src/main/bvmui/dist/index.html [hunk 1]
```diff
@@ -1 +1 @@
-<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><title>BK - Visual Manager</title><link rel="shortcut icon" href=favicon.ico><link href=/css/app.9f6a864a.css rel=preload as=style><link href=/js/app.cec374e4.js rel=preload as=script><link href=/js/chunk-vendors.8c726444.js rel=preload as=script><link href=/css/app.9f6a864a.css rel=stylesheet></head><body><noscript><strong>We're sorry but Bookkeeper Visual Manager doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.8c726444.js></script><script src=/js/app.cec374e4.js></script></body></html>
```
Comment (mino181295) [inline src/main/bvmui/dist/index.html]: I think we can drop the dist folder and add it to .gitignore

File: src/main/bvmui/package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "lodash": "4.17.15",
     "vue": "2.6.10",
     "vue-router": "^3.1.3",
-    "vuex": "3.1.1"
+    "vuex": "3.1.1",
+    "yarn": "^1.21.1"
```
Comment (mino181295) [inline src/main/bvmui/package.json]: Do we really need yarn?

File: src/main/bvmui/src/views/Cache.vue [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+            BookKeeper Visual Manager caches data on a local database in order to save resources on the Metadata Service (ZooKeeper).
             <br>
-            <a @click="refreshCache" >Refresh now !</a>
+            Last cache refresh was at <b>{{new Date(lastCacheRefresh)}}</b>.
+            <br>
+            Cache status <b>{{status}}</b> (<a @click="refreshPage" style="cursor: pointer;">Refresh</a>)
+            <br><br><br>
+            <a @click="refreshCache" style="cursor: pointer;">Reload now Bookie state and Ledger state</a>
+            <br><br><br>
+            <span>Current BookKeeper Client Configuration:</span>
```
Comment (mino181295) [inline src/main/bvmui/src/views/Cache.vue]: Nice

File: src/main/bvmui/package.json (hunk 2 above)
Comment (eolivelli) [reply src/main/bvmui/package.json]: it was added by yarn automatically, reverted

File: src/main/bvmui/dist/index.html (hunk 1 above)
Comment (eolivelli) [reply src/main/bvmui/dist/index.html]: there was a typo in .gitignore
fixed

**Pull request merged**